mobile/components/screens/Register.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { View, Text, TextInput, TouchableOpacity } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import Alert from '../utils/Alert';
+import LongLogo from "../../assets/longlogo.svg";
+import { validateRegisterInputs } from '../utils/Validator';
+
+const RegisterScreen = ({ navigation }) => {
+  const [formData, setFormData] = useState({
+    firstName: '',
+    lastName: '',
+    email: '',
+    password: '',
+    confirmPassword: '',
+  });
+
+  const [alertMessage, setAlertMessage] = useState(null);
+
+  const handleRegister = () => {
+    if (validateRegisterInputs(formData, setAlertMessage)) {
+      console.log("Rejestracja udana:", formData);
```

Comment:
te console logi bym wgl usunął następnym razem

Reply:
na ich miejsce jakaś logika sprawdzająca czy jest valid czy nie -> jak nie jest to `handleRegister` nie powinien pozwolić pójsć dalej